hetmech/hetmat.py

```diff
@@ -113,6 +114,30 @@ def read_first_matrix(specs):
         '\n'.join(paths))
 
 
+def _permute_matrix(adjacency_matrix, directed=False, multiplier=10,
+                    excluded_pair_set=set(), seed=0, log=False):
+
+    nonzeros = [list(i) for i in adjacency_matrix.nonzero()]
+    edge_list = list(zip(*nonzeros))
```

Comment:
Did it not work to just do:

```
list(zip(*adjacency_matrix.nonzero()))
```

i.e. not sure why we need to convert arrays to lists in line above, since we're just iterating through them anyways.